ci/Jenkinsfile.ios

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.9.24'
+
+/* Options section can't access functions in objects. */
+def isPRBuild = utils.isPRBuild()
+def isNightlyBuild = utils.isNightlyBuild()
+
+pipeline {
+
+  agent { label 'macos && arm64 && nix-2.24 && xcode-16.2' }
+
+  parameters {
+    booleanParam(
+      name: 'RELEASE',
+      description: 'Decides whether release credentials are used.',
+      defaultValue: params.RELEASE ?: false
+    )
+    choice(
+      name: 'VERBOSE',
+      description: 'Level of verbosity based on nimbus-build-system setup.',
+      choices: ['0','1','2','3']
+    )
+  }
+
+  options {
+    timestamps()
+    /* Prevent Jenkins jobs from running forever */
+    timeout(time: 45, unit: 'MINUTES')
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: '10',
+      daysToKeepStr: '30',
+      artifactNumToKeepStr: '3',
+    ))
+    /* Allows combined build to copy */
+    copyArtifactPermission('/status-desktop/*')
+    /* Abort old PR builds. */
+    disableConcurrentBuilds(abortPrevious: isPRBuild)
+    disableRestartFromStage()
+  }
+
+  environment {
+    GOCACHE = "${env.WORKSPACE_TMP}/go-cache"
+    GOMODCACHE = "${env.WORKSPACE_TMP}/go-mod-cache"
+    GOTMPDIR = "${env.WORKSPACE_TMP}"
+    PLATFORM = "ios/${getArch()}"
+    /* Improve make performance */
+    MAKEFLAGS = "-j4 V=${params.VERBOSE}"
+    QT_VERSION="6.9.2"
+    QMAKE = "/Users/admin/${QT_VERSION}/ios/bin/qmake"
+    QT_HOST_PATH = "/Users/admin/${QT_VERSION}/macos"
+    /* QMAKE = sh(script: "which qmake", returnStdout: true).trim() */
+    QTDIR = sh(script:"${env.QMAKE} -query QT_INSTALL_PREFIX", returnStdout: true).trim()
+    /* Enforce Go version installed infra-role-golang. */
+    /* to fix missing rcc, since QT6 rcc is located at ${QTDIR}/libexec/rcc */
+    PATH = "${env.QTDIR}/bin:${env.QTDIR}/libexec:${env.HOME}/go/bin:/usr/local/go/bin:${env.PATH}"
+    /* prevent sharing cache dir across different jobs */
+    GO_GENERATE_FAST_DIR = "${env.WORKSPACE_TMP}/go-generate-fast"
+    XDG_CACHE_HOME = "${env.WORKSPACE_TMP}/.cache"
+    /* iOS build configuration */
+    IPHONE_SDK = "iphoneos"
+    ARCH = "x86_64"
+    /* iOS app paths */
+    STATUS_IOS_APP_ARTIFACT = "pkg/Status-tablet-${BUILD_NUMBER}.zip"
+    STATUS_IOS_APP = "${WORKSPACE}/mobile/bin/ios/qt6/Status-tablet.app"
+  }
+
+  stages {
+    
+    stage('Fetch submodules') {
+      steps {
+        sh 'git submodule update --init --recursive'
+      }
+    }
+
+    stage('status-go') {
+      steps {
+        sh 'make status-go'
+      }
+    }
+
+    stage('Build iOS App') {
+      steps {
+        sh 'make mobile-build'
+      }
+    }
+
+    stage('Package iOS App') {
+      steps {
+        sh 'mkdir -p pkg'
+        sh "cd mobile/bin/ios/qt6 && zip -r ${env.WORKSPACE}/pkg/Status-tablet-${BUILD_NUMBER}.zip Status-tablet.app"
```

Comment:
Why are you not using `utils.pkgFilename()` like every other build?
https://github.com/status-im/status-desktop/blob/dd6489a1dac723aa63d5c30e1d979b5a33fcc6ee/ci/Jenkinsfile.linux#L83-L84
https://github.com/status-im/status-desktop/blob/dd6489a1dac723aa63d5c30e1d979b5a33fcc6ee/ci/Jenkinsfile.macos#L89
https://github.com/status-im/status-desktop/blob/dd6489a1dac723aa63d5c30e1d979b5a33fcc6ee/ci/Jenkinsfile.windows#L79-L81

Reply:
Also, why are we zipping it up?

Reply:
I made the filename end with `.app.zip` that way when you extract you get the proper structure with `.app` in the end